### PR TITLE
Feature/endring begrunnelse

### PR DESCRIPTION
**Pull request by ninakjek**
Viser kun perioder som trenger begrunnelse. 

Endringer frontend: https://github.com/navikt/familie-ba-sak-frontend/pull/281
Endringer i familie-ba-maler: https://github.com/navikt/familie-ba-dokgen/pull/46 
Favro: https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=Tea-2268

### CODE REVIEW

File: src/main/kotlin/no/nav/familie/ba/sak/behandling/fagsak/FagsakService.kt [hunk 1]
```diff
@@ -134,6 +136,11 @@ class FagsakService(
 
             val tilkjentYtelse = tilkjentYtelseRepository.findByBehandlingOptional(behandlingId = behandling.id)
 
+            val forrigeBehandling = behandlinger
```
Comment (henninghaakonsen) [inline src/main/kotlin/no/nav/familie/ba/sak/behandling/fagsak/FagsakService.kt:139]: Burde denne brukt hentForrige metoden?

File: src/main/kotlin/no/nav/familie/ba/sak/behandling/vilkår/VilkårController.kt [hunk 2]
```diff
@@ -73,7 +72,10 @@ class VilkårController(
         return ResponseEntity.ok(Ressurs.success(VilkårsvurderingUtils.hentVilkårsbegrunnelser()))
     }
 
-    fun settStegOgSlettUtbetalingBegrunnelser(behandlingId: Long) {
+    /**
+     * Når et vilkår vurderes (endres) vil begrunnelsene satt på dette vilkåret resettes
+     */
```
Comment (henninghaakonsen) [inline src/main/kotlin/no/nav/familie/ba/sak/behandling/vilkår/VilkårController.kt:77]: Bra

**Review by henninghaakonsen — APPROVED** [review]
image

File: src/main/kotlin/no/nav/familie/ba/sak/dokument/domene/maler/Innvilget.kt [hunk 3]
```diff
@@ -25,7 +25,7 @@ data class InnvilgetAutovedtak(
 
 data class DuFårSeksjon(
         val fom: String,
-        val tom: String,
+        val tom: String?,
```
Comment (ninakjek) [inline src/main/kotlin/no/nav/familie/ba/sak/dokument/domene/maler/Innvilget.kt:28]: Viser ikke til og med-biten i du-får-seksjonen hvis ikke: 
https://github.com/navikt/familie-ba-dokgen/pull/46 
https://github.com/navikt/familie-ba-sak/commit/5fe751f1f8abeca8e2312d4e634b17a5bc09b766#diff-8c84f7833a2b108843343c22e8d9ca4fR145

**Pull request merged**